src/sentry/static/sentry/app/components/group/externalIssues.jsx

```diff
@@ -41,13 +41,20 @@ class ExternalIssueForm extends AsyncComponent {
   renderBody() {
     let {integrationDetails} = this.state;
     let {action, group, integration} = this.props;
+    let config = integrationDetails[`${action}IssueConfig`];
+    let initialData = {};
+    config.forEach(field => {
+      // passing an empty array breaks multi select
```

Comment:
This shouldn't be the case and it definitely works outside of the `fieldFromConfig` implementation. What's the error?

Reply:
no error, just this and i can't remove them:
![screenshot 2018-06-04 17 52 01](https://user-images.githubusercontent.com/5026776/40948906-096af84e-6820-11e8-8a89-e41e984865df.png)

Reply:
since this pr is blocking some other stuff i want to get in and won't really affect our ability to start using the jira integration, i'd rather just file a ticket to come back to this

Reply:
Hmm, weird. Maybe just add a todo in the code to come back to, the workaround feels a bit hacky